### ISSUE TITLE
Univention Corporate Server (UCS)

### ISSUE THREAD

**Issue by Tentaculum** [junaid33/opensource.builders#374]
[Univention Corporate Server (UCS)](url) is a free-to-use platform designed to operate and manage server applications, user identities, and complete IT infrastructures. No matter whether you deploy Microsoft Windows, Mac OS X or Linux systems: UCS is optimally suited for the management of distributed, heterogeneous and virtualized IT environments,.
The integrated identity management system allows the  administration of users, identities and applications as well as the central management of server and desktop systems via an intuitive web interface. 
Furthermore, UCS users have free access to  the platform's own App Center which offers a large number of enterprise applications like Nextcloud, ownCloud, Collabora, ONLYOFFICE and many more. These applications can be integrated into the IDM of UCS and  be managed in a centralized way. too. All the downloaded  applications can be easily made available to users via an online portal and be accessed using single-sign-on. UCS offers AD functionality and an AD Connector to operate UCS parallel to a Windows Server.

**Features**

- Identity + Infrastructure management
- App Center: More than 90 applications for integration in UCS
- Active Directory: AD services via Samba 4, AD Connector + AD Takeover Wizzard
- IP and network administration: DNS Server BIND, DHCP, Webproxy/Squid
- Online Portal: Access to IT-services and applications for user via single-sign-on
- Printer management: CUPS
- File services: ownCloud, Nextcloud + Seafile
- System management: Open LDAP, Samba 4, AD Connector
- Monitoring: Nagios
- Mail and groupware: Mailstack with Potstfix + OX, Kopano, EGroupware
- APIs and REST API

**Comment by jcklpe** [comment]
@Tentaculum : Wrong format. The closed source software goes in the title of the issue with the open source alternatives in the comments. Marking as invalid. If not fixed in 30 days, the issue will be deleted.

**Comment by Tentaculum** [comment]
@jcklpe 
> @Tentaculum : Wrong format. The closed source software goes in the title of the issue with the open source alternatives in the comments. Marking as invalid. If not fixed in 30 days, the issue will be deleted.

Excuse my mistake; I'm completely new to GitHub. So, if I understand correctly, I'd have to put a closed software (Microsoft 365, for example) into the title - and then point out that UCS is an open source alternative to that in the comment section (which I already did)?
Thanks for your feedback!

**Comment by jcklpe** [comment]
@Tentaculum Yes thank you that is correct. You might also want to check to see if the closed software that it is an alternative to has already been submitted by searching the existing issues first too. For instance, Office 365 has already been posted with alternatives here: #63

**Comment by Tentaculum** [comment]
@jcklpe 
OK - slowly but surely I'm starting to get it :-) . So, I have seen that you have mentioned the issue in the Office 365 topic.
So, that means that Univention Corporate Server could be listed as an open source alternative there, once it is approved?
And, if that's correct also: is there anything I should do, given that this issue here is marked as unvalid? Should I re-enter the description in a comment there - or is your comment sufficient already?

Thanks againfor your feedback!

**Comment by jcklpe** [comment]
@Tentaculum : I think you misunderstand a little.   
  
The "mention" you see on that page is me linking that ticket in this ticket. Github tickets automatically interlink one another when they're mentioned. If I provide a link to a ticket, then that ticket will show in it's history that it has been mentioned elsewhere. It helps build up easy to follow threads between the web of tickets that make up a repo.   
  
Currently if you add your suggestion to that thread it will be approved in so far as it's a ticket where everyone is able to publicly post. If you want it to be added to the actual website then you'll need to make a change to the website, which is stored in this git repo and then do a pull request.  Once you do a pull request, a github bot will check it to make sure it builds properly, and then someone will check it and approve it to be merged into the master branch. Once it's merged into the master branch the github bot will automatically deploy it to the live production server.

**Comment by Tentaculum** [comment]
@ I think I got it now ;-) Thanks for your help. Edited the Title; the change to the website and the pull request will be made accordingly.
